.github/workflows/main.yml

```diff
@@ -535,11 +535,12 @@ jobs:
 
       - name: Push docs
         if: ${{ github.event_name == 'push' }}
-        uses: cpina/github-action-push-to-another-repository@main
+        uses: cpina/github-action-push-to-another-repository@v1.3
```

Comment:
what's the 1.3

Reply:
It's a tag, that honestly we probably should've been using before.

Reply:
Since `main` will pull the latest commit from the `main` branch every CI run.